nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py

```diff
@@ -203,6 +204,45 @@ def __init__(
         )
 
 
+class VietnameseCharsTokenizer(BaseCharsTokenizer):
+
+    _punctuation = '-();:,.!?¡¿—…"«»“”%'
+    _letters = 'abcdefghijklmnopqrstuvwxyzđáàãảạăắằẵẳặâấầẫẩậóòõỏọôốồỗổộơớờỡởợéèẽẻẹêếềễểệúùũủụưứừữửựíìĩỉịýỳỹỷỵ'
```

Comment:
could you please move `_punctuation` and `_letters` to `nemo/collections/common/tokenizers/text_to_speech/ipa_lexicon.py` accordingly, just following the same convention as `GermanCharsTokenizer`?